src/main/java/com/solvd/meta/carina/gui/pages/CartPage.java

```diff
@@ -0,0 +1,29 @@
+package com.solvd.meta.carina.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import com.solvd.meta.carina.gui.components.Header;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class CartPage extends AbstractPage {
+    @FindBy(xpath = "//button[@id='continue-shopping']")
+    private ExtendedWebElement continueShopping;
```

Comment:
Elements should be named according to their types. For example continueShopping -> continueShoppingButton
firstName -> firstNameField
Please check all elements and fix if needed